modules/packer/packer_test.go

```diff
@@ -117,3 +118,51 @@ func TestExtractArtifactINoIdPresent(t *testing.T) {
 		t.Error("Expected to get an error when extracting an Artifact ID from text with no Artifact ID in it, but got nil")
 	}
 }
+
+func TestFormatPackerArgs(t *testing.T) {
```

Comment:
Nice test! 👍

Reply:
Thanks again for the feedback! Here is a link to the output of my test run in AWS https://gist.github.com/floresj/9af415f4035f741a54bd1d22b83c888b

I initially started going the route of adding a `variables.json` to `terratest/examples/packer-basic-example/` but then decided to just dynamically generate that vars file within the go test in `test/packer_basic_example_test.go`. Since the `aws_region` is being randomly selected and used in subsequent calls, I figured that would keep it simple.